src/products/mozilla_org/testing.md

```diff
@@ -73,43 +73,32 @@ Updated translations are pushed to the production server regularly throughout th
+* Browser/Browsers in Firefox Browser/Browsers.
 
 If your community decides to translate any of these products or feature names, translate it once and translate it in this file. Be sure to have a consensus among active contributors what the translation should be before updating this file. This practice will ensure term use consistency across all pages.  
 
 Please refer to the [guidelines for brands and trademarks](https://mozilla-l10n.github.io/styleguides/mozilla_general/index.html#brands-copyright-and-trademark) when reviewing and updating this file.
 
-### Reviewing and updating migrated files
-
-The project dashboard contains fewer files than before. This list will grow over time as the mozilla.org team continues migrating from .lang to Fluent. Keep track of what is reviewed and what is pending for a review. Here are a few things to check:
-* On the project dashboard, if there is orange color in the status progress bar, click on the orange number of warnings. If the localized string is missing a term reference, chances are, the product or brand name is translated, transliterated or declensed. Insert the variable to replace the translated/transliterated/declensed term.
-* Do not translate words in placeables, like term references.
-* If there is enough bandwidth within a team, work in pairs, one updates the string as a suggestion, another reviews and approves it.
-* Review your changes on production, especially those with brand and product names.
-
 ### Reporting technical issues
 
 Report issues that can’t be resolved in Pontoon by:
-
 * Filing a bug in [Bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi?product=Mozilla%20Localizations) and selecting your locale.
 * Adding the l10n PM in charge for NI. The l10n PM information appears at the top left corner on the [project page](https://pontoon.mozilla.org/projects/mozillaorg/).
 * Including the file name plus a screenshot and/or URL.
+* If an extra tag is added to a string in order for it make sense in your language, the string will generate a linter error. The l10n PM for the project will consult you whether to make this an exception so it will not generate a linter error in the future.
 
 ### Reporting vendor translation errors
 
-If the mozilla.org project is not in your Pontoon dashboard, but the site is localized into your locale, it is likely being translated by Mozilla staff or a translation vendor. Report any translation errors from Mozilla staff or a translation vendor by:
+If the mozilla.org project is not in your Pontoon dashboard, but the site is localized into your locale, it is likely being translated by a translation vendor. Report any translation errors that can't be fixed in Pontoon by:
```

Comment:
```suggestion
If the mozilla.org project is not in your Pontoon dashboard, but the site is localized into your locale, it is likely being translated by a translation vendor. Report any translation errors by:
```

By definition (`If the mozilla.org project is not in your Pontoon dashboard`), it can't be fixed in Pontoon.